toqito/state_opt/state_exclusion.py

```diff
@@ -156,8 +223,70 @@ def _min_error_dual(
 
     # Objective function:
     problem.set_objective("max", picos.trace(y_var))
-    solution = problem.solve(solver=solver)
+    solution = problem.solve(solver=solver, **kwargs)
 
     measurements = [problem.get_constraint(k).dual for k in range(n)]
 
     return solution.value, measurements
+
+def _unambiguous_primal(
+    vectors: list[np.ndarray],
+    dim: int,
+    probs: list[float] = None,
+    solver: str = "cvxopt",
+    **kwargs,
+) -> tuple[float, list[picos.HermitianVariable]]:
+    """Solve the primal problem for unambiguous quantum state distinguishability SDP.
+
+    Implemented according to Equation (33) of arXiv:1306.4683.
```

Comment:
Add the `arxiv` reference to `articles.bib` and then cite it.

Reply:
Yeah, this I agree that we should cite from the bib (just not for the GitHub/GitLab issue).